exercises/sum.cpp

```diff
@@ -6,3 +6,32 @@
   Insert the second number: 2
   Sum: 3
 */
+
+//Program that calculate the sum of two number as input from user.
+
+#include <iostream>
+using namespace std;
+
+//declare the function that does the sum
+int sum(int a, int b)
+{
+    int s = a + b; //intialize the variable sum that will contain the sum
+    return s; //return the value of the sum
+}
```

Comment:
per risparmiare memoria potresti scrivere `return a + b;` in modo da evitare di creare `s`